src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Do not create new Random() inside method
![image](https://user-images.githubusercontent.com/20326858/232446644-cd36b5bb-4bd6-41f5-8609-99bef412c5de.png)